commands/admin/timeout.py

```
@@ -4,7 +4,8 @@
 
 import utility
 from localizer import tanjunLocalizer
-from utility import CommandInfo
+from utility import CommandInfo, EmbedColor
```

Flake8 (7.3.0)

[error] 'utility.CommandInfo' imported but unused

(F401)